client/public/index.css

```diff
@@ -0,0 +1,5 @@
+body{
```

Comment:
You should do this with `createGlobalstyle` in styled components: https://www.styled-components.com/docs/api. This can go in app.style.js.

<img width="765" alt="Screenshot 2019-03-24 at 13 04 57" src="https://user-images.githubusercontent.com/25960351/54877861-69657200-4e35-11e9-80c4-45d08189ae97.png">